nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -52,6 +59,7 @@ def __init__(
         group_size: int = None,
         ignored_scope: Optional[IgnoredScope] = None,
         all_layers: Optional[bool] = False,
+        mixed_precision_mode: Optional[MixedPrecisionMode] = MixedPrecisionMode.INT8_ERROR,
```

Comment:
`mixed_precision_mode` should be changed accordingly

Reply:
renamed